Atomic/syscontainers.py

```diff
@@ -923,7 +923,12 @@ def _get_containers_at(self, checkouts, are_preinstalled, containers=None):
             if fullpath.endswith(".0") or fullpath.endswith(".1"):
                 continue
 
-            with open(os.path.join(fullpath, "info"), "r") as info_file:
+            try:
+                fd = open(os.path.join(fullpath, "info"), "r")
+            except FileNotFoundError:
+                # Something got wrong?
+                continue
+            with fd as info_file:
```

Comment:
Why not just do a 
if not os.path.exists(os.path.join(fullpath, "info")):
           continue
with open ...